backend/tests/v5/e2e/routes/users.test.js

```diff
@@ -565,14 +565,14 @@ const testForgotPassword = () => {
 			expect(res.body.code).toEqual(templates.invalidArguments.code);
 		});
 
-		test('should not send email but return ok if user is an SSO user', async () => {
+		test('should send email and return ok if user is an SSO user', async () => {
 			await User.linkToSso(ssoTestUser.user, ssoTestUser.basicData.firstName, ssoTestUser.basicData.lastName,
 				ssoTestUser.basicData.email,
 				{ type: ServiceHelper.generateRandomString(), id: ServiceHelper.generateRandomString() });
 			await agent.post('/v5/user/password').send({ user: ssoTestUser.user })
 				.expect(templates.ok.status);
 			await User.unlinkFromSso(ssoTestUser.user, ssoTestUser.password);
-			expect(Mailer.sendEmail).not.toHaveBeenCalled();
+			expect(Mailer.sendEmail).toHaveBeenCalledTimes(1);
```

Comment:
we could also check the params (even though we dont do it for any test) ?

Reply:
I don't think it's necessary. it's a e2e test we just need to make sure it's sending an email.

Too much effort 😆